sdk/monitorexporter/azure-analytics-monitorexporter/pom.xml

```diff
@@ -0,0 +1,78 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
can the sdk/monitorexporter/ directory just be sdk/monitor/ ? we will be adding other Monitor SDKs in the future and they should live near the OT-exporter

Reply:
@srnagar Are we planning to add more monitor exporters in the future then I would suggest keeping the `sdk/monitorexporter/`?

Reply:
Moved this to `sdk/monitor` to be consistent with where other languages are hosting this library. If we need a separate directory, we can safely move it as that is not a breaking change.